### PR TITLE
Rupak

### PR DESCRIPTION
**Pull request by Rupak1017**
Completed content updates requested by Amil on the Home Page, AI page, Our People1 page, Our people2 page(Amil's Story), and Publication Page. Please refer to this document for Amil's changes on 07/27/24 https://docs.google.com/document/d/1isZiE1mJKFrgvvVvrW5qFeaQ5LkImQ2QB671ezzeFrs/edit?hl=en

Content changes on home page
image 

Advisor name updation zoom out view 
image

Alt image updation and content changes of Amil's Story.
image

AI Page h1 change
image

Publication page h2 change
image

### CODE REVIEW

**Review by ridham-virufy — DISMISSED** [review]

**Review by ridham-virufy — COMMENTED** [review]
Why is the Join Our team button on the publications page going to share your cough module? can you fix that error? @Rupak1017

**Review by a-wong-8 — COMMENTED** [review]
Need to change the join out team button to correct link. And add question mark to "Interested to collaborate on our research"

File: src/i18n/en/publications.ts [hunk 1]
```diff
@@ -42,8 +42,8 @@ export const Publications = {
     },
   ],
   coughSection: {
-    title: "Let's help the world and save lives by recording your cough",
+    title: "Interested to collaborate on our research",
```
Comment (a-wong-8) [inline src/i18n/en/publications.ts]: question mark needed here

**Review by ridham-virufy — APPROVED** [review]
Good Work

**Pull request closed without merging**